### ISSUE TITLE
Next/Prev Card convenience methods

### ISSUE THREAD

**Issue by jesseflorig** [ontehfritz/www.mtgdb.info#41]
Not sure if this is a legit issue, but I'm putting it here as a backend feature request.

Something like Card.NextCard & Card.PrevCard that returns a full card object based on the card number in the set (not multiverse id). So then I could use Card.NextCard.Name and Card.NextCard.Id

**Comment by ontehfritz** [comment]
I will look into a solution for this.

**Comment by ontehfritz** [comment]
I made a assumption that you wanted this for card view, I added this to the CardModel, not to the actual Card object, but if it is looking like we need it in multiple places then I will. You will have to check for null on PrevCard or NextCard in the razor view. If the number is messed up for some reason. This will make it apparent and we will fix the data.  This will be in my next commit tonight (April 12)
